plugins/TodayAtMun/today.py

```diff
@@ -0,0 +1,53 @@
+from datetime import datetime, timedelta
+from json import load
+from pathlib import Path
+from typing import Dict
+
+
+class Today:
+    """A class used to go find significant days on Mun Calendar"""
```

Comment:
Few things here:
* Docstring does not need to specify this is a class
* Incorrect capitalization of "MUN"
* Missing trailing period